test/use-swr.test.tsx

```diff
@@ -1854,17 +1858,16 @@ describe('useSWR - suspense', () => {
       })
       return <div>{data}</div>
     }
-    const { container } = render(
+    render(
       <Suspense fallback={<div>fallback</div>}>
         <Section />
       </Suspense>
     )
 
-    // hydration
-    expect(container.textContent).toMatchInlineSnapshot(`"hello"`)
+    await screen.findByText('hello')
   })
-
   it('should throw errors', async () => {
+    jest.spyOn(console, 'error').mockImplementation(() => {})
```

Comment:
i am not sure mocking console.errror here is a good idea, but it does give us cleaner output in console
![cleanoutput.png](https://vip1.loli.net/2020/11/18/UlLDwCjZmnPczQh.png)